params/config.go

```diff
@@ -993,7 +993,7 @@ func (c *ChainConfig) Rules(num *big.Int, isMerge bool, timestamp uint64) Rules
 		IsBerlin:         c.IsBerlin(num),
 		IsLondon:         c.IsLondon(num),
 		IsMerge:          isMerge,
-		IsShanghai:       c.IsShanghai(timestamp),
+		IsShanghai:       currentArbosVersion >= 11,
```

Comment:
We should still have an IsShanghi func since it's public, it should receive both arbos and timestamp, and should check if it's arbitrum (see IsLondon)

Reply:
done.